client/components/Username.vue

```diff
@@ -1,6 +1,11 @@
 <template>
 	<span
-		:class="['user', {[nickColor]: store.state.settings.coloredNicks}, {active: active}]"
+		:class="[
```

Comment:
this should probably be a data attribute, not a class. Same as related fields

Reply:
I'll switch it to a data attribute. Thanks!